english/marshmallow.md

```diff
@@ -20,6 +20,9 @@ The type of extreme mortal danger that so often accompanies confectionary inspir
 Quickly surveying your immediate surrounds, you notice a garishly dressed, 
 vacant-eyed young man lounging on a nearby sofa.
 
+[pinch yourself to see if this is a dream](./is-this-a-dream/is-this-a-dream.md)
+[remember that you are superman](./remember-you-are-superman/remember-you-are-superman.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.